crates/starknet-server/src/api/json_rpc/error.rs

```diff
@@ -25,7 +25,7 @@ pub enum ApiError {
     InvalidTransactionIndexInBlock,
     #[error("Class hash not found")]
     ClassHashNotFound,
-    #[error("Contract error: {error}")]
+    #[error("Contract error")]
```

Comment:
According to the specification, shouldn't there also be some `data` in this error: https://github.com/starkware-libs/starknet-specs/blob/49665932a97f8fdef7ac5869755d2858c5e3a687/api/starknet_api_openrpc.json#L3903-L3916

Otherwise we are again hiding the real cause of error and making it hard for users to debug. Maybe this can be handled in a separate PR?

Reply:
the users should use the data property because the spec defines only "Contract error" as data to the `message` field

Reply:
Mistake on my end, didn't see where ContractError is converted to RpcError. Resolving.